.github/workflows/mb_funcs.yml

```
@@ -0,0 +1,16 @@
+name: Morebits function tests
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    paths:
    - 'morebits.js'
  pull_request:
    paths:
    - 'morebits.js'
```

No need to trigger on every PR, just the ones that change morebits. Per https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths

---

Dope!

---

Mmm, what about adding `- 'test/**`?  Might as well run the tests if tests change...

---

Yep, was about to say that!